Source/WebKit/Shared/UpdateInfo.h

```diff
@@ -52,7 +52,7 @@ class UpdateInfo {
 
     // The size of the web view.
     WebCore::IntSize viewSize;
-    float deviceScaleFactor;
+    float deviceScaleFactor { 0.0f };
```

Comment:
WebKit doesn't allow this style.
https://webkit.org/code-style-guidelines/#float-suffixes

Reply:
that better?

Reply:
See https://webkit.org/code-style-guidelines/#float-suffixes

Reply:
I force pushed the PR to use this, was that not right?:
```c
float deviceScaleFactor { 0 };
```